statsig_rum/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "86a69c7d-8042-4f93-96b1-f139b2058644",
+  "app_id": "statsig-rum",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Enrich your Datadog RUM data with feature gate information from Statsig",
+    "title": "Statsig - RUM",
+    "media": [{
+      "media_type": "image",
+      "caption": "Datadog Feature Flag Dash",
```

Comment:
One last comment - can you update these captions to be a bit more descriptive and specific to each image?